lib/plug/conn.ex

```diff
@@ -987,7 +1095,9 @@ defmodule Plug.Conn do
   It accepts the same options as `read_body/2`.
   """
   @spec read_part_body(t, Keyword.t()) :: {:ok, binary, t} | {:more, binary, t} | {:done, t}
-  def read_part_body(%{adapter: {adapter, state}} = conn, opts) do
+  def read_part_body(conn, opts)
```

Comment:
Is this change required? There's only a single clause for the `read_part_body` function